### DISCUSSION TITLE
AddDisjunction for group of Nodes

### DISCUSSION THREAD

**Discussion by harvetech** [google/or-tools#4156] [Routing (and legacy CP) questions]
Dear Or-Tools community,

 How can I add a disjunction not for nodes but for group of nodes. e.g [ [A,B], [C,D], [E,F] ]. The task is to at max two group of nodes can be active. Means one group of node has to be dropped. I know how to add disjunction between two nodes as per the example **routing.AddDisjunction([node1, node2],1)** , How to do the same for the group of nodes ? any suggestion would greatly help me. 

Best Regards

**Comment by FirstPotatoOnMars** [comment]
You can just force the nodes in the same group to the same vehicle, then dropping one of the group would lead to the drop of the whole group of nodes.    `routing.solver().Add(routing.VehicleVar(A) == routing.VehicleVar(B))`
